### ISSUE TITLE
ViroCore 1.18.0, ARCore 1.21.0

### ISSUE THREAD

**Issue by iTruff** [viromedia/virocore#346]
I was able to update ARCore version to 1.21.0 and build new virocore-release-v_1_18_0.aar.
See files below:

[core-1.21.0.aar.zip](https://github.com/viromedia/virocore/files/5549790/core-1.21.0.aar.zip)
[virocore-release-v_1_17_0.aar.zip](https://github.com/viromedia/virocore/files/5549792/virocore-release-v_1_17_0.aar.zip)

**Comment by hannesa2** [comment]
I want to see it on this (or any master)
How can I help to automate this ?

**Comment by iTruff** [comment]
Hi @hannesa2 Unfortunately I can't make it public due to lack of time.  

You can do it yourself:
[How to build ViroCore](https://github.com/viromedia/virocore#building-the-virocore-platform)
[How to update ARCore version](https://github.com/viromedia/virocore/blob/master/android/libs/arcore/README.txt)

**Comment by ebelinski** [comment]
@iTruff When I use these two AAR files, I encounter this error:

`More than one file was found with OS independent path 'lib/armeabi-v7a/libarcore_sdk_c.so'. If you are using jniLibs and CMake IMPORTED targets, see https://developer.android.com/studio/preview/features#automatic_packaging_of_prebuilt_dependencies_used_by_cmake`

Could it be that you uploaded the wrong AAR file for ViroCore? I noticed your text says "virocore-release-v_1_18_0.aar" but the file below is "virocore-release-v_1_17_0.aar.zip".

**Comment by iTruff** [comment]
@ebelinski No, it's just a naming issue, you can fix your problem with custom packaging options in your build gradle

```
packagingOptions {
        pickFirst 'lib/armeabi-v7a/libarcore_sdk_c.so'
        pickFirst 'lib/arm64-v8a/libarcore_sdk_c.so'
        pickFirst 'lib/x86_64/libarcore_sdk_c.so'
    }
```
